src/utils/money.cljs

```diff
@@ -0,0 +1,55 @@
+(ns utils.money
```

Comment:
FYI: This namespace is temporarily being duplicated because I needed a function from the "old" code to format large amounts, but the linter fails if I require any namespace from `status_im` inside `quo2`.

I'm committed to work on a follow-up PR to move the entire `money` namespace to the new code structure since it's used in numerous places, it's worth it. As soon as this PR is merged ;)